v6/scripts/release.sh

```
@@ -0,0 +1,39 @@
+)
+
+response=$(
+    curl -H "$AUTH" \
+    --fail \
+    --silent \
+    --location \
+    --data "$payload" \
+    "https://api.github.com/repos/${REPO}/releases"
+)
+upresponse=$(curl -sH "$AUTH" $GH_TAGS)
+eval $(echo "$upresponse" | grep -m 1 "id.:" | grep -w id | tr : = | tr -cd '[[:alnum:]]=')
```

Shellcheck

[warning] Quote this to prevent word splitting.

(SC2046)